docs/source/installation/installation_docker.rst

```diff
@@ -17,9 +17,45 @@ Note that you will need to publish the port in order to access it from
 the host, or other machines. For more information, read
 https://docs.docker.com/engine/reference/run/
 
-Note that the default latest tag will typically be a development version
-and may not be very stable. You can specify a tagged version to run a
-release instead:
+To access a Postgres database running on a localhost you need to use
```

Comment:
A subheading for local-postgres would be good here since not everyone needs to know about that